test/test_poolmanager.py

```diff
@@ -259,6 +259,23 @@ def test_http_connection_from_context_case_insensitive(self) -> None:
         assert pool is other_pool
         assert all(isinstance(key, PoolKey) for key in p.pools.keys())
 
+    def test_deprecated_no_scheme(self) -> None:
+        p = PoolManager()
+
+        with pytest.warns(DeprecationWarning) as records:
+            p.request(method="GET", url="evil.com://good.com")
+
+        msg = (
+            "URLs without a scheme (ie 'https://') are deprecated and begin raising an error "
```

Comment:
Message will have to be updated here too

Reply:
Done ✔️